downstream/assemblies/platform/assembly-installing-controller-operator-external-db.adoc

```diff
@@ -39,7 +39,7 @@ Once you have configured your {ControllerName} operator, click *Create* at the b
 [role="_additional-resources"]
 == Additional resources
 
-* See link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.2/html/operators/index[Working with Operators in OpenShift Container Platform] for more information on running operators on OpenShift Container Platform.
+* For more information on running operators on OpenShift Container Platform, navigate to the link:https://access.redhat.com/documentation/en-us/openshift_container_platform/[OpenShift Container Platform product documentation] and click the `Operators - Working with Operators in OpenShift Container Platform` guide.
```

Comment:
```suggestion
* For more information on running operators on OpenShift Container Platform, navigate to the link:https://access.redhat.com/documentation/en-us/openshift_container_platform/[OpenShift Container Platform product documentation] and click the _Operators - Working with Operators in OpenShift Container Platform_ guide.
```

I think references to Guide titles should be in italics. See "Reference to Red Hat guides" in https://redhat-documentation.github.io/asciidoc-markup-conventions/